test/integration/spec/localtracks.js

```diff
@@ -75,3 +89,119 @@ const createLocalTracks = require('../../../lib/createlocaltrack');
     });
   });
 });
+
+describe('LocalParticipant', function() {
```

Comment:
```suggestion
describe('LocalTrackPublication', function() {
```

Reply:
@makarandp0 This test should move to `test/integration/spec/localtrackpublication.js`.